logic/scorer/scorer.go

```diff
@@ -91,33 +85,30 @@ func scoreFifteens(ptVals values) (scoreType, int) {
 		return none, 0
 	}
 
-	var numFifteens uint
+	var numFifteens int
```

Comment:
:cactus: I think we had this as a uint because then it signaled that `fifteen0 << numFifteens` would never accidentally have a negative shift

Reply:
We have that on one hand vs. having to cast to `int` on the other hand. If `numFifteens` was negative it'd be a developer error (not runtime), so I felt safe changing it... I'm pretty :shrug: on it though